app/src/main/java/de/westnordost/streetcomplete/quests/moped/AddProhibitedForMoped.kt

```diff
@@ -0,0 +1,49 @@
+package de.westnordost.streetcomplete.quests.moped
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddProhibitedForMoped : OsmFilterQuestType<ProhibitedForMopedAnswer>() {
+
+    override val elementFilter =
+        // filter all cycleways next to a road with a speed of more than 50, as Moped use is designated by law
+        """
+        ways with (
+            highway = cycleway
+            or (cycleway and bicycle ~ yes|designated and (maxspeed <= 30 or !maxspeed))
```

Comment:
1. What about `highway = path` (or `footway`) + `bicycle = designated`? A bike path can also be mapped like this.
2. What's that `cycleway` doing there? It looks wrong.

Reply:
1. A moped is only allowed on a bike path. If it is shared with pedestrians, it isn't allowed
2. If I understand correctly, there are 2 types of bike path tags. https://wiki.openstreetmap.org/wiki/Key:cycleway 
Highway = cycleway -> separate way/road for bikes
cycleway = * (e.g. lane) -> part of road to be used by bikes

Therefore I added the 'cycleway'  tag on roads with a speed limit of 50.
Maybe I can remove this filter since in this case it doesn't matter whether you're riding on the cycleway or a road next to it?

Reply:
> A moped is only allowed on a bike path. If it is shared with pedestrians, it isn't allowed

Segregated foot and bike paths can be tagged like `highway=path` + `segregated=yes` + `foot=designated` + `bicycle=designated` or also `highway=footway` + `segregated=yes` + `bicycle=designated` or also `highway=cycleway` + `segregated=yes` + `foot=designated`.
Conversely, `highway=cycleway` alone does not guarantee that it is a path that is segregated from pedestrians, only if `segregated=yes` is set it is.

> cycleway = * (e.g. lane) -> part of road to be used by bikes

Yes, but it is much more complicated than that. First of all, there is also `cycleway:both`, `cycleway:left` and `cycleway:right`. Second, the situation can differ per side. So, this would require a different interface that lets you select each side individually. Plus, you would have to handle situations where it is defined for one side, but not for the other, etc.
This would be a lot of effort, I recommend just not asking for cycleways that are mapped as properties on roads. It looks like most cycleways in Belgium are mapped separately anyway.